src/qililab/digital/circuit_transpiler.py

```diff
@@ -142,8 +154,8 @@ def transpile_circuits(
     def route_circuit(
         self,
         circuit: Circuit,
-        placer: Placer | type[Placer] | tuple[type[Placer], dict] | None = None,
-        router: Router | type[Router] | tuple[type[Router], dict] | None = None,
+        placer: Union["Placer", type["Placer"], tuple[type["Placer"], dict], None] = None,
+        router: Union["Router", type["Router"], tuple[type["Router"], dict], None] = None,
```

Comment:
same here, the previous syntax will work with the `from __future__ import annotations`.